css/index.css

```diff
@@ -0,0 +1,79 @@
+@media screen and (min-width: 480px) {
+
+
+	html {
+		display: flex;
+		flex-flow: row wrap;
+		justify-content: space-around;
+	
+		background-color: black;
+		font-family: "Times New Roman", Times, serif;
+		color: green;
+		padding: 1px;
+		margin: 1px;
+	}
+}
+
+header {
+    background-color: gray;
+    color: white;
+    padding: 1px;
+}
+
+body {
+    background-color: #f8f8ff;
+}
+
+section {
+
+}
+
+section ul {
+
+}
+
+a {
+
+}
+
+a:hover {
+ 
+}
+
+h2 {
+	color: orange;
+	font-size: 16pt;
+}
```

Comment:
One of the tasks was to change font, color _and weight_ of **all headings**. You updated `h2` headings, but not the `h1` headings. An easy way to apply the same CSS to multiple elements / types is to just add the additional heading to the declaration, so `h1, h2 { }`. 

In addition, the `font-weight` property can be changed on these headers, as well.